deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-console-stack.ts

```diff
@@ -23,6 +23,7 @@ import { OtelCollectorSidecar } from "./migration-otel-collector-sidecar";
 export interface MigrationConsoleProps extends StackPropsExt {
     readonly migrationsSolutionVersion: string,
     readonly vpc: IVpc,
+    readonly mskEnabled?: boolean,
     readonly streamingSourceType: StreamingSourceType,
```

Comment:
Let's use StreamingSourceType.DISABLED in place of mskEnabled==false in the stack here. We should start to think about BYO Kafka and that enum to fit naturally for this purpose

Reply:
oh, wow, that was already there and I totally missed it. Updated.